lib/tsubaki/matchers/validate_corporate_number_of_matcher.rb

```diff
@@ -37,10 +42,11 @@ def failure_message
 
         def description
```

Comment:
I'm not sure whet's this method for, but `<<` would not work with `# frozen_string_literal: true`